TeachEasy/src/teacheasy/data/ButtonData.java

```diff
@@ -0,0 +1,85 @@
+package teacheasy.data;
+/**
+ * @version 1.0 05 Feb 2015
+ * @author sh1157 and sr896
+ */
+

```

Comment:
Style: Version and Author @annotations should be included in the class documentation comment below.